rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "stable"
+components = ["rustfmt", "clippy"]
+profile = "minimal"
```

Comment:
https://rust-lang.github.io/rustup/concepts/profiles.html
Shouldn't we just use profile = "default"?